tests/02-compile-arm-ports-01/Makefile

```diff
@@ -38,8 +38,8 @@ hello-world/cc2538dk \
 rpl-border-router/cc2538dk \
 rpl-border-router/cc2538dk:MAKE_ROUTING=MAKE_ROUTING_RPL_CLASSIC \
 hello-world/nrf52dk \
-platform-specific/nrf52dk/coap-demo/nrf52dk:coap-server \
-platform-specific/nrf52dk/coap-demo/nrf52dk:coap-client:SERVER_IPV6_EP=ffff \
+platform-specific/nrf52dk/coap-demo/nrf52dk/coap-server \
+platform-specific/nrf52dk/coap-demo/nrf52dk/coap-client:SERVER_IPV6_EP=ffff \
```

Comment:
I think we wanted `:` here as these are compile options. I believe this is what causes the test to hand

Reply:
That's actually one of the changes done by this PR -  `coap-server` and `coap-client` them to different directories.

My build script does not support building a single Makefile with multiple config options. Given how the Contiki build system works, it's usually a terrible idea to have source files in a single directory for executables that need different configurations (since the system files are build just once even if the Makefile sets two different system configurations for the two executables).

So, in my mind it's better to change this example to consistent with the rest of Contiki examples.

Reply:
Right, but then the problem was that there is a missing `/nrf52dk` at the end, so CI was attempting to build for platform `coap-server` instead. Fixed in https://github.com/atiselsts/contiki-ng/pull/1